.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+**Motivation**:
+
+Explain why you're making this change and what problem you're trying to solve.
```

Comment:
The guide text is better to show as a comment. `<!-- -->`. The deletion will occur too many times to do that.